private/bufpkg/bufimage/bufimagetesting/bufimagetesting_test.go

```diff
@@ -451,11 +453,9 @@ func TestBasic(t *testing.T) {
 		},
 		newImage.Files(),
 	)
-	require.Equal(
-		t,
-		protoImage,
-		bufimage.ImageToProtoImage(newImage),
-	)
```

Comment:
This assertion was failing after the change. But the only thing I spotted were differences in internal unexported bits of the proto struct. So changing to do proper proto comparison fixed it.